poule.yml

```diff
@@ -33,13 +33,6 @@
                 platform/arm:        [ "raspberry", "raspbian", "rpi", "beaglebone", "pine64" ],
             }
         }
-      - type:       version-label
```

Comment:
while people still report docker issues, I think we should keep the `version-label`